components/autoComplete/autoCompleteItem.js

```diff
@@ -28,12 +28,12 @@ class AutoCompleteItem extends Component {
     const {
       children,
       dataAttrs = {},
-      mods = [],
       onClick,
       isActive,
       isTitle,
       ...otherProps
     } = this.props;
+    const mods = this.props.mods ? this.props.mods.slice() : [];
```

Comment:
I commented something similar in another PR, but seems to me that the following could be a good convention

    getClassNamesWithMods('ui-button', [
      isTitle && 'title',
      isActive && 'active',
    ]);